### PR TITLE
Updates access levels to make sense

### PR DESCRIPTION
**Pull request by Majkl-J**
## About The Pull Request

Gives doctors coroner access on lowpop.
Also gives security medics back medical access.

## Changelog

:cl:
qol: Doctor access levels now make sense
/:cl:

### CODE REVIEW

File: code/datums/id_trim/jobs.dm [hunk 1]
```diff
@@ -670,6 +670,7 @@
 	extra_access = list(
 		ACCESS_PLUMBING,
 		ACCESS_VIROLOGY,
+		ACCESS_MORGUE_SECURE,
```
Comment (nevimer) [inline code/datums/id_trim/jobs.dm:673]: one comment

**Review by nevimer — CHANGES_REQUESTED** [review]
Sure. Looks good. Just comment this line with a bubber addition

**Comment by nevimer** [comment]
This will go poof next upstream update, but, we can have nice things until then.

**Comment by Majkl-J** [comment]
> This will go poof next upstream update, but, we can have nice things until then.

Ill just PR it again.

**Pull request merged**